interop-cats/jvm/src/main/scala/scalaz/zio/interop/catsjvm.scala

```diff
@@ -30,53 +30,69 @@ abstract class CatsPlatform extends CatsInstances {
 }
 
 abstract class CatsInstances extends CatsInstances1 {
-  implicit def ioContextShift[E]: ContextShift[IO[E, ?]] = new ContextShift[IO[E, ?]] {
-    override def shift: IO[E, Unit] =
-      IO.yieldNow
+  implicit def ioContextShift[R]: ContextShift[TaskR[R, ?]] = new ContextShift[TaskR[R, ?]] {
+    override def shift: TaskR[R, Unit] =
+      ZIO.yieldNow
 
-    override def evalOn[A](ec: ExecutionContext)(fa: IO[E, A]): IO[E, A] =
+    override def evalOn[A](ec: ExecutionContext)(fa: TaskR[R, A]): TaskR[R, A] =
       fa.on(ec)
   }
 
-  implicit def ioTimer[R <: Clock, E]: effect.Timer[ZIO[R, E, ?]] = new effect.Timer[ZIO[R, E, ?]] {
-    override def clock: cats.effect.Clock[ZIO[R, E, ?]] = new effect.Clock[ZIO[R, E, ?]] {
-      override def monotonic(unit: TimeUnit): ZIO[R, E, Long] =
+  implicit def ioTimer[R <: Clock]: effect.Timer[TaskR[R, ?]] = new effect.Timer[TaskR[R, ?]] {
+    override def clock: cats.effect.Clock[TaskR[R, ?]] = new effect.Clock[TaskR[R, ?]] {
+      override def monotonic(unit: TimeUnit): TaskR[R, Long] =
         zioClock.nanoTime.map(unit.convert(_, NANOSECONDS))
 
-      override def realTime(unit: TimeUnit): ZIO[R, E, Long] =
+      override def realTime(unit: TimeUnit): TaskR[R, Long] =
         zioClock.currentTime(unit)
     }
 
-    override def sleep(duration: FiniteDuration): ZIO[R, E, Unit] =
+    override def sleep(duration: FiniteDuration): TaskR[R, Unit] =
       zioClock.sleep(scalaz.zio.duration.Duration.fromNanos(duration.toNanos))
   }
 
-  implicit val taskEffectInstances: effect.ConcurrentEffect[Task] with SemigroupK[Task] =
-    new CatsConcurrentEffect
+  implicit def taskEffectInstances[R](implicit runtime: Runtime[R]): effect.ConcurrentEffect[TaskR[R, ?]] =
+    new CatsConcurrentEffect[R](runtime)
 
-  implicit val taskParallelInstance: Parallel[Task, Util.Par] =
-    parallelInstance(taskEffectInstances)
 }
 
 sealed abstract class CatsInstances1 extends CatsInstances2 {
-  implicit def ioMonoidInstances[E: Monoid]: MonadError[IO[E, ?], E] with Bifunctor[IO] with Alternative[IO[E, ?]] =
-    new CatsAlternative[E] with CatsBifunctor
+  implicit def ioMonoidInstances[R, E: Monoid]
+    : MonadError[ZIO[R, E, ?], E] with Bifunctor[ZIO[R, ?, ?]] with Alternative[ZIO[R, E, ?]] =
+    new CatsAlternative[R, E] with CatsBifunctor[R]
 
-  implicit def parallelInstance[E](implicit M: Monad[IO[E, ?]]): Parallel[IO[E, ?], ParIO[E, ?]] =
-    new CatsParallel[E](M)
+  implicit def taskConcurrentInstances[R]: effect.Concurrent[TaskR[R, ?]] =
```

Comment:
`with SemigroupK[Task]` since that's available

Reply:
Not `TaskR[R, ?]` here?

Reply:
Sorry, fixed